src/object.d

```diff
@@ -4956,3 +4956,315 @@ string _d_assert_fail(string comp, A, B)(A a, B b) @nogc @safe nothrow pure
     buffer[n .. n + valB.length] = valB;
     return (() @trusted => cast(string) buffer)();
 }
+
+// These are needed to fake the safety level to `@safe pure nothrow`.
+// Without this code will break as these hooks where previously exported
+// as having this safety level.
+T[] _d_arrayappendT(T)(ref T[] x, scope T[] y) @trusted
+{
+    alias func(T) = T[] function(ref T[] x, scope T[] y) @trusted pure nothrow;
+    auto f = cast(func!T)&_d_arrayappendT_!T;
+    return f(x, y);
+}
+
+T[] _d_arrayappendcTX(T)(ref T[] px, size_t n) @trusted
+{
+    alias func(T) = T[] function(ref T[] px, size_t n) @trusted pure nothrow;
+    auto f = cast(func!T)&_d_arrayappendcTX_!T;
+    return f(px, n);
+}
+
+T[] _d_arrayappendcT(T)(ref T[] px, scope T y) @trusted
+{
+    import core.internal.traits : Unqual;
+    import core.stdc.string : memcpy;
+    auto result = _d_arrayappendcTX(px, 1);
+    memcpy(cast(Unqual!T *)&result[$ - 1], cast(Unqual!T *)&y, T.sizeof);
+    return result;
+}
+
+/**
+ * Run postblit on `t` if it is a struct and needs it.
+ * Or if `t` is a array, run it on the children if they have a postblit.
+ */
+void __doPostblit(T)(auto ref T t)
```

Comment:
Can this be `private` in this module?  Also, why the `__` prefix?

Reply:
Yes, I can add private. The `__` is because it called this inside rt.lifetime: https://github.com/dlang/druntime/blob/master/src/rt/lifetime.d#L701